src/axe-runner.js

```diff
@@ -65,12 +65,11 @@ function processReports(reports) {
         `${sumViolations} accessibility violations have been detected in total.`
       )
     );
+    process.exit(1);
   } else {
     log(chalk.green('No accessibility violations found in any packages'));
+    process.exit(0);
   }
-
-  if (violationsDetected) process.exit(1);
-  process.exit(0);
```

Comment:
Why wasn't this working? :D

Reply:
Didn't we simply want to change this for `sumViolations`?

Reply:
`violationsDetected` was an array and supposed to be checked against its length. I overlooked the mishap and never tested this eventuality again.

Reply:
Not sure what that would mean as exit code means, but maybe even `process.exit(sumViolations)`?

Reply:
I'd keep terminating to process with exitCode = 1, I've got no prediction how Travis will handle exitCode > 1.